include/psa/crypto_config.h

```diff
@@ -757,13 +757,13 @@
 /**
  * \def MBEDTLS_NIST_KW_C
  *
- * Enable the Key Wrapping mode for 128-bit block ciphers,
- * as defined in NIST SP 800-38F. Only KW and KWP modes
- * are supported. At the moment, only AES is approved by NIST.
+ * Enable the Key Wrapping mode as defined in NIST SP 800-38F. Currently only
+ * KW and KWP modes, with AES and PSA_ALG_ECB_NO_PADDING are supported by the
+ * library.
```

Comment:
ECB is an implementation detail. It isn't relevant here.

The wording is also a little weird because KW and KWP are all the modes that can be used with AES, anyway. (SP 800-38F defines three modes, and the third mode is only for 64-bit block ciphers, which means DES in NIST specifications.)

Also, preexisting: these modes are often known from the RFC rather than from NIST, so it would be helpful to cite the RFC here as well.

```suggestion
 * Enable the 128-bit key wrapping modes from NIST SP 800-38F:
 * KW (also known as RFC 3394) and KWP (RFC 5649).
 * Currently these modes are only supported with AES.
```